docs/developers-guide/devenv.md

```diff
@@ -141,6 +141,34 @@ And of course your Jetty development server is available via
 clojure -M:run
 ```
 
+You can also start a REPL another way (e.g., through your editor) and then call
+
+```
+(require 'dev)
```

Comment:
The `dev/user.clj` file has a handy function: `(dev)` which has source:

```clojure
(defn dev
  "Load and switch to the 'dev' namespace."
  []
  (require 'dev)
  (in-ns 'dev)
  :loaded)
```

I just do `(do (dev) (start!))` and it starts up the full application.